client/signup/style.scss

```diff
@@ -447,15 +447,15 @@ body.is-section-signup.is-white-signup .layout:not( .dops ):not( .is-wccom-oauth
 			.formatted-header__title,
 			.formatted-header__subtitle {
 				padding: 0;
-				text-align: left;
+				text-align: start;
```

Comment:
Running on local Calypso instance I'm getting `text-align: right;` for these selections, coming from http://calypso.localhost:3000/calypso/evergreen/entry-stepper.rtl.css with the selector being `.design-setup .design-picker .formatted-header .formatted-header__subtitle`.  It looks like there are lot of different places where `.formatted-header .formatted-header__subtitle` selector is used, should they all be updated?

Reply:
Here're the screenshots of how it looks on my end:
<img width="1783" alt="Screenshot 2022-06-29 at 14 35 24" src="https://user-images.githubusercontent.com/7847633/176437826-55002398-563a-4e1b-bf8d-e438c9fa4493.png">
<img width="1783" alt="Screenshot 2022-06-29 at 14 36 02" src="https://user-images.githubusercontent.com/7847633/176437853-3b4d5fee-aed1-4ab8-a220-d5332888666e.png">

Reply:
The latest commit should fix this issue. Overall, this PR focuses on the issues mentioned in 437-gh-Automattic/i18n-issues. We can use this method to fix similar issues as they arise in the future, or create a new ticket where we remove our dependency on the webpack-rtl-plugin.